src/components/Menu/MenuItem.tsx

```diff
@@ -96,7 +110,12 @@ class MenuItem extends UIComponent<IMenuItemProps, any> {
     const { children, content } = this.props
 
     return (
-      <ElementType {...rest} className={classes.root} onClick={this.handleClick}>
+      <ElementType
+        {...rest}
+        className={classes.root}
+        onClick={this.handleClick}
+        {...this.accBehavior.generateAriaAttributes(this.props, this.state)}
```

Comment:
`...rest` needs to come after each acc behavior spread so it can be overridden.  Let's add a conformance test that ensures users can override acc props.